vita3k/gui/src/game_selector.cpp

```diff
@@ -15,18 +15,50 @@
 // with this program; if not, write to the Free Software Foundation, Inc.,
 // 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
+#include "private.h"
+
 #include <gui/functions.h>
 #include <gui/state.h>
 
-#include "private.h"
-
 #include <host/state.h>
+
+#include <util/log.h>
 #include <util/string_utils.h>
 
 using namespace std::string_literals;
 
 namespace gui {
 
+bool refresh_game_list(GuiState &gui, HostState &host) {
+    auto game_list_size = gui.game_selector.games.size();
+
+    if (!gui.game_selector.games.empty())
```

Comment:
```suggestion
```
no

Reply:
Why no ? I no want clear if game clear is already empty 🤔

Reply:
Calling `clear()` on an empty map will just have it return :)